README.md

```diff
@@ -94,6 +94,19 @@ The following settings affect the interpretation of JSON responses:  (You must s
                       :response-format (raw-response-format)
                       :timeout 100}))
 
+; Send multiple files explicitly, ClojureScript specific
+; input-element is an html element of file type.
+(let [form-data (let [f-d (js/FormData.)
+                      files (.-files input-element)
+                      name (.-name input-element)]
+                    (do
```

Comment:
Sorry, still haven't looked at this. Will tomorrow night, I promise. :(

On Tuesday, 19 January 2016, ASHISH NEGI <notifications@github.com> wrote:

> In README.md
> <https://github.com/JulianBirch/cljs-ajax/pull/123#discussion_r50139211>:
>
> > @@ -94,6 +94,19 @@ The following settings affect the interpretation of JSON responses:  (You must s
> >                        :response-format (raw-response-format)
> >                        :timeout 100}))
> >
> > +; Send multiple files explicitly, ClojureScript specific
> > +; input-element is an html element of file type.
> > +(let [form-data (let [f-d (js/FormData.)
> > +                      files (.-files input-element)
> > +                      name (.-name input-element)]
> > +                    (do
>
> Just wanted to be explicit. :)
> I have removed that, please check.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JulianBirch/cljs-ajax/pull/123/files#r50139211>.
>


-- 
Sent from an iPhone, please excuse brevity and typos.